Compilation errors

Hello,

I had a fresh install of ```picongpu@develop``` with gcc version 9.3.0, ```CUDA 11.1``` and ```openmpi 4.0.5``` and there are some compilation errors for my model when using PIConGPU as ```spack load picongpu +adios %gcc@9.3.0 && export PIC_BACKEND="cuda:75"```. 

The first error was:
```include/picongpu/param/particle.param:9:10: fatal error: pmacc/nvidia/rng/distributions/Uniform_float.hpp: No such file or directory``` so I disabled the header in ```particle.para``` since I don't seem to make use of it but the second error came as ```spack/linux-linuxmint20-westmere/gcc-9.3.0/picongpu-develop-xihuo4szr2tdyxw5p5gx6scr7n3omjtp/include/pmacc/../pmacc/nvidia/atomic.hpp(95): error #304: no instance of function template "alpaka::warp::activemask" matches the argument list```. I attach to whole model as a folder where you can find the ```out.txt``` and ```out_2.txt``` compilation log files. 

I also tried using the stable PIConGPU version but there were some ```gcc``` compiler incompatibilities and I switched to ```picongpu@develop```

Maybe you can help me to debug this model, please
Thank you.
Cristian

[LAYERS-2D-PLATES.zip](https://github.com/ComputationalRadiationPhysics/picongpu/files/5385630/LAYERS-2D-PLATES.zip)

I still have some problems with the new installation. In Spack I have ```picongpu 0.4.3``` installed with ```gcc 7.5.0``` from the system and also on the system ```CUDA 11.1``` and ```openmpi 4.0.5``` to be used by ```picongpu```

I fetch the LWFA mode and compile like:

![Screenshot_2020-10-21_21-58-37](https://user-images.githubusercontent.com/7108691/96786744-9a7a9f80-13e8-11eb-93be-657b1c7d9c87.png)

and I get an error like ```spack/opt/spack/linux-linuxmint20-westmere/gcc-7.5.0/picongpu-0.4.3-nvff4ydogmuyq62tzaqzaplyfianjo72/thirdParty/alpaka/include/alpaka/core/ConcurrentExecPool.hpp:361:19: error: base operand of ‘->’ is not a pointer```.

The whole log file is attached.

[out.txt](https://github.com/ComputationalRadiationPhysics/picongpu/files/5418706/out.txt)

I would like to use the lastest version that is ```picongp@develop``` or ```version 5.0``` but I don't know what is the right compiler for it.

> I still have some problems with the new installation. In Spack I have `picongpu 0.4.3` installed with `gcc 7.5.0` from the system and also on the system `CUDA 11.1` and `openmpi 4.0.5` to be used by `picongpu`
> 
> I fetch the LWFA mode and compile like:
> 
> ![Screenshot_2020-10-21_21-58-37](https://user-images.githubusercontent.com/7108691/96786744-9a7a9f80-13e8-11eb-93be-657b1c7d9c87.png)
> 
> and I get an error like `spack/opt/spack/linux-linuxmint20-westmere/gcc-7.5.0/picongpu-0.4.3-nvff4ydogmuyq62tzaqzaplyfianjo72/thirdParty/alpaka/include/alpaka/core/ConcurrentExecPool.hpp:361:19: error: base operand of ‘->’ is not a pointer`.
> 
> The whole log file is attached.
> 
> [out.txt](https://github.com/ComputationalRadiationPhysics/picongpu/files/5418706/out.txt)
> 
> I would like to use the lastest version that is `picongp@develop` or `version 5.0` but I don't know what is the right compiler for it.

The error looks like a compiler issue or a combination of CUDA11 +gcc7.5. The code is definitive correct.

> I would like to use the lastest version that is picongp@develop or version 5.0 but I don't know what is the right compiler for it.
Yuu can use any C++11 compiler for 0.5.0 and any C++14 compiler for the development branch. ALl gcc 6+ schould work in both cases.

--- a/etc/picongpu/bash/mpiexec.tpl
+++ b/etc/picongpu/bash/mpiexec.tpl
@@ -53,7 +53,7 @@ export OMPI_MCA_io=^ompio
 if [ -f !TBG_dstPath/input/bin/cuda_memtest ] ; then
   mpiexec -am !TBG_dstPath/tbg/openib.conf --mca mpi_leave_pinned 0 -npernode !TBG_gpusPerNode -n !TBG_tasks !TBG_dstPath/input/bin/cuda_memtest.sh
 else
-  echo Note: GPU memory test was skipped as no binary 'cuda_memtest' available. This does not affect PIConGPU, starting it now" >&2
+  echo "Note: GPU memory test was skipped as no binary 'cuda_memtest' available. This does not affect PIConGPU, starting it now" >&2
 fi
 
 if [ $? -eq 0 ] ; then

